vhdbuilder/packer/install-dependencies.sh

```diff
@@ -472,34 +465,22 @@ ls -ltr /usr/local/bin/* >> ${VHD_LOGS_FILEPATH}
 # v1.16.13-hotfix.20200714.1
 # v1.17.7-hotfix.20200714.1
 # v1.17.9-hotfix.20200714.1
-# v1.18.4-hotfix.20200624.1
-# v1.18.6
-PATCHED_HYPERKUBE_IMAGES="
+# v1.18.4-hotfix.20200626.1
+# v1.18.6-hotfix.20200723.1	
+K8S_VERSIONS="
 1.14.7-hotfix.20200408.1
 1.14.8-hotfix.20200529.1
-1.15.7-hotfix.20200326
-1.15.10-hotfix.20200408.1
-1.15.11-hotfix.20200529.1
-1.15.11-hotfix.20200714.1
-1.15.12-hotfix.20200623.1
-1.15.12-hotfix.20200714.1
-1.16.7-hotfix.20200601.1
-1.16.8.1
-1.16.9-hotfix.20200529.1
-1.16.10-hotfix.20200623.1
-1.16.10-hotfix.20200714.1
-1.16.13-hotfix.20200714.1
-1.17.3-hotfix.20200601.1
-1.17.4
-1.17.5.1
-1.17.7-hotfix.20200624
-1.17.7-hotfix.20200714.1
-1.17.9-hotfix.20200714.1
-1.18.1.1
-1.18.2-hotfix.20200624
-1.18.4-hotfix.20200624
-1.18.4-hotfix.20200624.1
```

Comment:
Please keep these as we havent' updated our toggle yet

1.18.4-hotfix.20200624	
1.18.4-hotfix.20200624.1

Reply:
https://msazure.visualstudio.com/CloudNativeCompute/_git/aks-config?path=%2Ftoggles%2Fhyperkube-images-need-patch.yaml&_a=contents&version=GBmaster